src/OrchardCore.Modules/OrchardCore.Tenants/Views/Admin/Index.cshtml

```diff
@@ -200,9 +203,33 @@
     </ul>
 </form>
 
+<div class="row">
+<div class="col-11">
 @await DisplayAsync(Model.Pager)
+</div>
+<div class="col-1 my-3">
+@{
+    var query = new Dictionary<string, StringValues>(Context.Request.Query);
+    query.Remove("PageSize");
+    var queryString = QueryString.Create(query);
+    var pageSizes = new string[]{"10", "25","50", "100"};
```

Comment:
Instead of defining this here we could add it to the Model.Pager shape. Model.Pager.PageSizes and Model.Pager.DefaultPageSize.

Reply:
Don't hesitate to contribute to the PR if you want.

Reply:
@Skrypt Would you please help me achieving the remaining tasks?